### PR TITLE
docs: Azure AD/SAML integration

### PR DESCRIPTION
**Pull request by norefice-github**
Added ad saml user add

### CODE REVIEW

File: content/how-to/add-user-ad-saml/index.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+14. Select the BluBracket application
+![select blubracket application](blubracketapplication2.png)
+15. Copy and save the Application (client) ID
+![application client id](applicationclientid.png)
+16. Navigate to https://TENANT_NAME.blubracket.com/settings/auth (Be sure to replace TENANT_NAME with the name of your tenant name provided by BluBracket)
+    1. Expand Single Sign-On and enter the Federation metadata document URL from step 13.
+![blubracket sso](blubracketsso.png)
+17. Enter the Application (client) ID from step 15
+![blubracket authentication](blubracketclientid.png)
+18. Tenant administrators will be required to add the desired Azure AD users to their BluBracket tenant prior to login via Single Sign-On. (https://TENANT_NAME.blubracket.com/settings/users) (Be sure to replace TENANT_NAME with the name of your tenant name provided by BluBracket)
```
Comment (misterbisson) [inline content/how-to/add-user-ad-saml/index.md:64]: Is this step really part of the 19 step list, or is it a separate task?

File: content/how-to/add-user-ad-saml/index.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+- src: enterpriseapplications.png
+---
+
+This article covers the SAML 2.0 authentication requests and responses that BluBracket supports for Single Sign-On (SSO) using Azure Active Directory (Azure AD).
+
+BluBracket requires an email attribute in order to successful identify and authenticate users; BluBracket looks for this vaule from the SAML identity claim http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress.
+
+As of the last update to this article users must exists in BluBracket before they are able to login successfully using Single Sign-On.  BluBracket does not automatically register new users that do not previously exist in the tenant.
+
+1. Login to Azure AD as an Administrator [https://portal.azure.com](https://portal.azure.com)
```
Comment (misterbisson) [inline content/how-to/add-user-ad-saml/index.md:20]: Is this really a single 19 step task, or is it a set of smaller tasks? It looks like:

1. Register a new Azure Active Directory app
2. Grant permissions to users in Azure AD to use BluBracket
3. Specify the Azure AD connection details in BluBracket
4. Add users in the BluBracket portal

File: content/how-to/add-user-ad-saml/index.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+weight: 300
+toc: true
+resources:
+- src: azureactivedirectory.png
+- src: enterpriseapplications.png
+---
+
+This article covers the SAML 2.0 authentication requests and responses that BluBracket supports for Single Sign-On (SSO) using Azure Active Directory (Azure AD).
+
+BluBracket requires an email attribute in order to successful identify and authenticate users; BluBracket looks for this vaule from the SAML identity claim http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress.
```
Comment (misterbisson) [inline content/how-to/add-user-ad-saml/index.md:16]: Could this be said more clearly if the document started with an overview of the logical relationship between Azure AD and BluBracket? Key points appear to be:

1. Users must be registered in both Azure AD and BluBracket in order to use BluBracket—removing a user from either will terminate their access to BluBracket
2. The email address in BluBracket must match the email address in Azure AD
3. Passwords and 2fa are managed in Azure ID

File: content/how-to/add-user-ad-saml/index.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+title: "Adding a BluBracket user with Azure AD SAML"
+description: "How to add a BluBracket user with Azure AD SAML"
+lead: "How to integrate with Azure AD SAML"
+date: 2022-01-12T02:48:57+00:00
+draft: false
+weight: 300
+toc: true
+resources:
+- src: azureactivedirectory.png
+- src: enterpriseapplications.png
```
Comment (misterbisson) [inline content/how-to/add-user-ad-saml/index.md:11]: Only a couple images specified here, but lots of images in the commit

**Review by robbyandrews — APPROVED** [review]

**Pull request merged**